pep-0649.rst

```diff
@@ -727,8 +731,8 @@ The defined ``format`` values are guaranteed to be contiguous,
 and the ``inspect`` module also publishes attributes representing
 the minimum and maximum supported ``format`` values::
 
-    FORMAT_MIN = VALUE
-    FORMAT_MAX = SOURCE
+    FORMAT_MIN = AnnotationFormat.VALUE
+    FORMAT_MAX = AnnotationFormat.SOURCE
```

Comment:
If now an enum, do these still make sense, as one could just iterate through the enum?

Reply:
I also don't see a strong use case for these, so maybe we should just drop them. We can always re-add them in a later version if a good use case comes up.

Reply:
Removed.  Iterating through the enum is an even better solution.

Reply:
(work is in other PR)

Reply:
#3138